### PR TITLE
feat: Add an option to override item favorite blocker

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java [hunk 1]
```diff
@@ -33,22 +35,49 @@ public class ItemFavoriteFeature extends Feature {
...
+
+        if (lootChestCloseOverride.get() == 0) {
+            McUtils.sendMessageToClient(Component.translatable("feature.wynntils.itemFavorite.closingBlocked")
+                    .withStyle(ChatFormatting.RED));
+            e.setCanceled(true);
+            return;
+        }
+
+        if (lootChestCloseOverride.get() < lootChestCloseOverrideCounter) return;
+        lootChestCloseOverrideCounter++;
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java:69]: These two lines seem to be in the wrong order..? Have you tried that this work?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java:69]: Yes, it does. "If the current override counter is higher than the set minimum config, do not try to block closing".

Comment (magicus) [reply common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java:69]: Ah, right, the  comparison was between the config max to the left and the actual counter to the right.

For some reason I found that counter-intuitive. Do you mind fliipping the comparison to a > instead?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java:69]: Did, no problem.

**Review by magicus — APPROVED** [review]

**Pull request merged**